static/js/lightbox.js

```diff
@@ -39,7 +57,7 @@ function display_image(payload, options) {
 
         $("#lightbox_overlay .image-preview").html(canvas).show();
         var photo = new LightboxCanvas(canvas);
-        photo.speed(2.3);
+        photo.speed(2.3 * lightbox_settings.zoom_speed);
```

Comment:
there's no control for zoom speed yet, right?  I think we should avoid adding a setting for zoom speed.  It seems much more likely to be that the zoom speed is just broken on firefox and we need to fix that issue, not that different people prefer widely different zoom speeds.